rampwf/utils/pretty_print.py

```diff
@@ -2,13 +2,14 @@
 """
 Utility methods to print the results in a terminal using term colors
 """
-import platform
+import os
 
 from pandas import option_context
 
 from ..externals.colored import stylize, fg, attr
 
-IS_WINDOWS = platform.system() == "Windows"
+COLOR_TERMS = ['xterm-256color', 'cygwin', 'xterm-color']
```

Comment:
```suggestion
COLOR_TERMS = ['xterm-256color', 'cygwin', 'xterm-color', 'xterm']
```

to make it work on my macos

Reply:
Ah, that's a problem. I added 'xterm' first but it's also 'xterm' in the anaconda terminal on windows which has no handling of the escape sequences. On my mac (both in Terminal and in iterm) it's 'xterm-256color'. What should we do? Be more conservative or liberal?

Reply:
I could make a more complex condition:
```
'TERM' in os.environ and (os.environ['TERM'] in COLOR_TERMS or (os.environ['TERM'] == 'xterm' and not IS_WINDOWS)
```